sdcm/utils/common.py

```diff
@@ -48,7 +48,7 @@
 
 LOGGER = logging.getLogger('utils')
 DEFAULT_AWS_REGION = "eu-west-1"
-AwsArchType = Literal['x86_64', 'arm64']
+AwsArchType = ['x86_64', 'arm64']
```

Comment:
It's used for type hints only. Please, just remove this line and replace it's usage with just `str`. (btw, it'll solve the problem w/ pylint too)

Reply:
i can indeed remove it completely, but the thing is, it won't solve the problem of `TypeError: 'ec2.Image' object is not subscriptable` because now the `ec2` object is created by a different function (`filter()` instead of `describe_images()`)...
i will fix the unittests as well